Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Installation stage
+# Offical Node.js Alpine base image
+FROM node:lts-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json for utilising Docker cache 
+# to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies only needed
+RUN yarn install --production --frozen-lockfile
+
+# Copy installed dependencies to current working directory
+COPY . .
+
+# Build stage
+FROM base AS builder
+ENV NODE_ENV=production
+WORKDIR /build
+COPY --from=base /base ./
+RUN yarn add --dev typescript @types/react @types/node && yarn build
```

Comment:
It seems like you've added the `devDependencies` here because of `--production` in `yarn install` command. Should we remove the `--production` so this `RUN` command only has `yarn build`?

Reply:
First, I used only `yarn build` command but it cause an error and failed to build image. If you would like to see the detail, you can check in our Dockerhub repository.